astropy/modeling/core.py

```diff
@@ -1272,7 +1272,7 @@ def parameters(self, value):
         stop = self._param_metrics[self.param_names[-1]]["slice"].stop
 
         try:
-            value = np.array(value).flatten()
+            value = np.array(value).ravel()
```

Comment:
If `value` is already an array, this will still cause a copy, while below the value is copied again to the parameters. I'd write `np.asanyarray(value).ravel()`

Reply:
Done. Thanks!